### PR TITLE
chore: fix return type covariance issues

### PR DESCRIPTION
**Pull request by mussbach**
Phpstan complained about invalid covariant method signatures (https://www.php.net/manual/en/language.oop5.variance.php)

### How to review/test
Code review and use phpstan to validate 

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly

### CODE REVIEW

File: demosplan/DemosPlanStatementBundle/Logic/AssessmentExportOptions.php [hunk 1]
```diff
@@ -270,10 +270,7 @@ public function get($section)
         return $this->options[$section];
     }
 
-    /**
-     * @return mixed
-     */
-    public function jsonSerialize()
+    public function jsonSerialize(): Collection
```
Comment (AlexMenschoid) [inline demosplan/DemosPlanStatementBundle/Logic/AssessmentExportOptions.php:273]: Either I'm missing something or this is very wrong. Why should jsonSerialize return a Collection? Especially when it is used in another method that tries to json encode that collection. I think the naming is throwing me for a loop here but I would prefer the naming to get fixed together with the return type.

Comment (mussbach) [reply demosplan/DemosPlanStatementBundle/Logic/AssessmentExportOptions.php:273]: This is just how the current implementation works. I did not want to change anything (yet)

Comment (mussbach) [reply demosplan/DemosPlanStatementBundle/Logic/AssessmentExportOptions.php:273]: @AlexMenschoid may I merge?

Comment (mussbach) [reply demosplan/DemosPlanStatementBundle/Logic/AssessmentExportOptions.php:273]: ping @AlexMenschoid

**Pull request merged**